include/revng/Pipeline/Step.h

```diff
@@ -62,6 +68,24 @@ class Step {
   const ContainerSet &containers() const { return Containers; }
   ContainerSet &containers() { return Containers; }
 
+  void setArtifacts(const Kind *ArtifactsKind, std::string ContainerName) {
+    this->Artifacts.Kind = ArtifactsKind;
```

Comment:
this-> is only mandatory to access fields presents in base classes, here it can be written as Artifcats.Kind